libraries/chain/include/eosio/chain/transaction.hpp

```diff
@@ -117,7 +116,7 @@ namespace eosio { namespace chain {
       explicit packed_transaction(signed_transaction&& t, compression_type _compression = none)
       :signatures(std::move(t.signatures))
       {
-         set_transaction(t, std::move(t.context_free_data), _compression);
+         set_transaction(t, t.context_free_data, _compression);
```

Comment:
It does anyway. `set_transaction` takes it as a const& and changing that would not help because it is packed and that is a "copy" anyway. I have further refactored this in #6471 to actually do a move.

Reply:
Oh okay. Well I was going to say we should fix `set_transaction` to also have an overload that takes an rvalue, but if you are going to refactor it in #6471, then this is okay as is.